src/overlays/actors/ovl_En_Diving_Game/z_en_diving_game.c

```diff
@@ -199,8 +199,8 @@ void EnDivingGame_Talk(EnDivingGame* this, PlayState* play) {
                 }
             }
         } else {
-            if (Text_GetFaceReaction(play, 0x1D) != 0) {
-                this->actor.textId = Text_GetFaceReaction(play, 0x1D);
+            if (MaskReaction_GetTextId(play, MASK_REACTION_SET_ZORA) != 0) {
```

Comment:
it might be out of scope for this PR but this `!= 0` feels like its begging for a `!= TEXT_ID_NONE` or `TEXT_ID_INVALID` define, so many of these actors are using it